src/js/common/components/authentication/LoginPrompt.jsx

```diff
@@ -45,10 +45,14 @@ class LoginPrompt extends React.Component {
               <li>View Post-9/11 GI Bill enrollment information</li>
             </ul>
             <p>
-              <button className="usa-button-primary va-button-primary usa-button-big" onClick={this.handleLogin}><strong>Sign In</strong></button>
-              <button className="va-button-secondary usa-button-big" onClick={this.handleSignup}><strong>Create Account</strong></button>
+              <button className="usa-button-primary va-button-primary usa-button-big" onClick={this.handleLogin}><strong>Log in</strong></button>
+              <button className="va-button-secondary usa-button-big" onClick={this.handleSignup}><strong>Sign up</strong></button>
             </p>
             <p>Even without an account, you can apply for VA benefits.</p>
+            <h5>Having trouble signing in?</h5>
```

Comment:
I wonder why we're changing the language from "Sign In" to "Log In" but using "signing in" here.